### ISSUE TITLE
How to build TfLiteMobilenetQuantized-Armnn

### ISSUE THREAD

**Issue by angolini** [ARM-software/armnn#134]
How to build the test TfLiteMobilenetQuantized-Armnn?

I have this CMake flags I think is related to this test:
-DBUILD_TF_LITE_PARSER=1
-DBUILD_ARMNN_EXAMPLES=1
-DBUILD_TESTS=1

What else would be needed? Some external package, maybe?

Thanks in advance

**Comment by MatthewARM** [comment]
Hi @angolini to build the TfLite parser you need the flatbuffers library, and the generated flatbuffer schema files from TfLite. There are instructions for getting those things here: https://developer.arm.com/technologies/machine-learning-on-arm/developer-material/how-to-guides/configuring-the-arm-nn-sdk-build-environment-for-tensorflow-lite

Basically on your cmake line you also need to set `-DFLATBUFFERS_ROOT=<something>
-DTF_LITE_GENERATED_PATH=<something>`

**Comment by angolini** [comment]
I think I setup both PATCH properly, the code now finds flatbuffer.h but I face a build error due to mismatch of definition:

`error: no matching function for call to 'flatbuffers::Verifier::Verify(const flatbuffers::Vector<flatbuffers::Offset<tflite::SubGraph> >*)'`

I have used find to find all the flatbuffers.h I have available, and all of them are the same, and I cannot spot where this mismatch could be coming from. Would this error ring a bell for you?

**Comment by MatthewARM** [comment]
Hi @angolini did it work in the end? Was our documentation wrong?

Many thanks,
Matthew

**Comment by angolini** [comment]
Hi @MatthewARM I got it working, eventually. The documentation helped me a lot! However, I only got to know the documentation because you've sent it to me. (And I went to the ARM documentation base, I only did not realized I needed TfLite documentation (and not the other)

I still don't understand the flatbuffers mismatch. I was downloading the TfLite 1.11, which I eventually figured out depends on flatbuffers 1.10.0. But I got it working with 1.9.0, the same version I've got the error with.

long story short: it worked!

Thanks for your contact and help.

**Comment by MatthewARM** [comment]
OK thanks for the feedback! I'll look in to making the documentation links clearer.
